docs/usage/cli/filter.rst

```diff
@@ -63,3 +71,184 @@ The filter command allows you to select a specific number of sequences from spec
     --output filtered.fasta
 
 This subsampling and filtering will reduce the number of sequences in the tutorial data set from 34 to 24.
+
+Subsampling beyond the capabilities of ``augur filter``
+-------------------------------------------------------
+
+There are some subsampling strategies in which a single call to ``augur filter``
+does not suffice. One such strategy is "tiered subsampling". In this strategy,
+mutually exclusive sets of filters, each representing a "tier", are sampled with
+different subsampling rules. This is commonly used to create geographic tiers.
+Consider this subsampling scheme:
+
+    Sample 20 sequences from North America and 50 sequences from Europe.
```

Comment:
This is a random example that I made up. Open to other suggestions.

Reply:
I think this is fine. An alternative would be early vs late. But the general aim of this is to introduce multiple rounds of `augur filter`, and presupposes people are already familiar with the capabilities of an individual command.

Reply:
This is an interesting example because it exposes a potential issue with the term "tiered" which implies some hierarchical structure that isn't actually present in this example (North America and Europe both being at the same geographic resolution).

Another common example is to select all strains from a given division and then contextual samples from the rest of the country and other regions like @trvrb's example in a comment on [the related section of the subsampling design doc](https://docs.google.com/document/d/10jrk4OaLhfo6io8WjtZuZFD9BDpr85GtghO1x5GAOM8/edit#heading=h.unjwv0rt29ja). You could use the "all data from Washington plus some global context" example in these docs or borrow from [the nice example of this kind of geographic "tiered" subsampling you made for the ncov tutorial](https://docs.nextstrain.org/projects/ncov/en/latest/tutorial/genomic-surveillance.html#break-down-the-command), @victorlin. Either of these examples sets us up for a subsequent example of proximity-based sampling with the division-specific sample as the "focal" sample.

Reply:
@huddlej Good point. I wanted to make two points here:

1. There are many types of subsampling that cannot be done in a single call to ``augur filter``.
2. Tiered subsampling is one common practical example.

Then I created a simple example which isn't tiered. It's probably better to use an example that is requires a slightly more complicated query but represents tiered subsampling. How about this:

> Sample 100 sequences from Washington State and 50 sequences from the rest of the world.

Reply:
I chatted with @huddlej in person and we settled on this example:

> Sample 100 sequences from Washington state and 50 sequences from the rest of the United States.

Done in f070dbe67d4c08434f42009d5950c7d878883b85.